jmix-ui/ui/src/main/java/io/jmix/ui/xml/layout/loader/FileMultiUploadFieldLoader.java

```diff
@@ -120,4 +123,19 @@ protected void loadPasteZone(UploadField uploadField, Element element) {
             }
         }
     }
+
+    protected void loadTotalProgressDisplayEnabled(FileMultiUploadField uploadField, Element element) {
+        String totalProgressDisplayEnabled = element.attributeValue("totalProgressDisplayEnabled");
+        if (StringUtils.isNotEmpty(totalProgressDisplayEnabled)) {
```

Comment:
Unfortunately, we prefer `guave.Strings` over `StringUtils`, so `!Strings.isNullOrEmpty`